Transfer/Sources/CreateTransferPresenter.swift

```diff
@@ -348,6 +349,8 @@ final class CreateTransferPresenter {
             return
         }
 
+        view.dismissKeyboard()
```

Comment:
Why need this here?

Reply:
While validating fields when transfer button tapped, table view is getting reloaded. That time amount text field gets the keyboard focus. It entered amount is valid, will show spinner before moving to confirm screen. There keyboard is visible with spinner. 
![Simulator Screen Shot - iPhone 11 - 2020-11-03 at 12 56 46](https://user-images.githubusercontent.com/71116197/98019708-18e33280-1e28-11eb-972f-1f7981998d46.png)

Reply:
In that case remove `textField.becomeFirstResponder()` from TransferAmountCell rather than adding this more logic

Reply:
Yes, this causes adding more code to remove the keyboard and reloads, hence will remove `textField.becomeFirstResponder()`. So by default, keyboard will not shown when user enters this page.